website/blog/2022-01-21-components-and-blocks.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Components and blocks
+description: Explanation of the difference between components and blocks on a practical example
+slug: components-and-blocks
+authors: iobrado
+tags: [eightshift, boilerplate, components, blocks]
+image: #
+hide_table_of_contents: false
+---
+In this blog post, we'll give you a practical explanation about the difference between components and blocks. For a truly in-depth explanation about components and blocks, you can refer to [our documentation](https://infinum.github.io/eightshift-docs/docs/basics/blocks). More specific explanation about component in a block can be found in [Component in a Block](https://infinum.github.io/eightshift-docs/docs/basics/blocks-component-in-block/) section.
+
+<!--truncate-->
+One of the things that is really confusing to newcomers is the difference between components and blocks. Let's break it all down.
+
+### Component
+
+**A component is like a blank slate, a template without context**. It doesn't know or even care for what it will be used. Can you combine multiple simple components into one complex component? Absolutely! That's exactly what we'll do in the following example. Let's say we have a component which consists of an image and a heading.  Image and heading are two simple components that come with the theme out-of-the-box.
```

Comment:
```suggestion
**A component is like a blank slate, a template without context**. It doesn't know or even cares for what it will be used. Can you combine multiple simple components into one complex component? Absolutely! That's exactly what we'll do in the following example. 

Let's say we have a component that consists of an image and a heading. Image and heading are two simple components that come with the theme out-of-the-box.
```